### PR TITLE
skip validate option

### LINKED ISSUE #1147: feat: Parse file without running any validations
### ACH Version

v1.26.1

### What were you trying to do?

I would like to be able to parse an ACH file without running any validations on the records. Often files are so badly formatted that working through validation errors causes a lot of overhead. 

### What did you expect to see?

A `ValidateOpts` option to disable all validations.

### PR DESCRIPTION
**Pull request by mfdeveloper508**
- SkipAllValidate into ValidateOpts
- skip all validate logic of reader when the option is enable 

Fixes: https://github.com/moov-io/ach/issues/1147

### CODE REVIEW

**Comment by adamdecaf** [comment]
Can we add some tests? This is a pretty major change.

**Comment by mfdeveloper508** [comment]
> Can we add some tests? This is a pretty major change.

sure, I will add soon

**Comment by mfdeveloper508** [comment]
office is off in this week? I don't see any coworker in slack channel

**Comment by adamdecaf** [comment]
Can we rebase off `origin/master` when the tests are added? Thanks

File: file.go [hunk 1]
```diff
@@ -678,6 +678,9 @@ type ValidateOpts struct {
 
 	// UnequalAddendaCounts skips checking that Addenda Count fields match their expected and computed values.
 	UnequalAddendaCounts bool `json:"unequalAddendaCounts"`
+
+	// skipAllValidate can be set to disable all of validate actions that included im
+	SkipAllValidate bool `json:"skipAllValidate"`
```
Comment (yordis) [inline file.go]:
```suggestion
	SkipAllValidation bool `json:"skipAllValidation"`
```

Would that be a better name?

**Comment by adamdecaf** [comment]
Why does this PR include changes that are already on `master`?

**Comment by mfdeveloper508** [comment]
skip-validation option will be run on file/reader's level, not record's level

File: file.go [hunk 2]
```diff
@@ -679,8 +679,8 @@ type ValidateOpts struct {
 	// UnequalAddendaCounts skips checking that Addenda Count fields match their expected and computed values.
 	UnequalAddendaCounts bool `json:"unequalAddendaCounts"`
 
-	// skipAllValidate can be set to disable all of validate actions that included im
-	SkipAllValidate bool `json:"skipAllValidate"`
+	// SkipAllValidation can be set to disable all of validate actions
+	SkipAllValidation bool `json:"skipAllValidate"`
```
Comment (yordis) [inline file.go]:
```suggestion
	SkipAllValidation bool `json:"skipAllValidation"`
```

**Review by adamdecaf — APPROVED** [review]

**Pull request merged**